neat/utils.py

```diff
@@ -62,7 +62,7 @@ def create_prediction_map(genomes, dataset, config):
         phenotype.to(config.DEVICE)
         for input in dataset:
             input.to(config.DEVICE)
-            prediction = phenotype(input)
+            prediction = phenotype(input).to('cpu')
```

Comment:
should  this be `.to('cpu')`? Seems like that disallows any GPU acceleration in the future. `config` supplied has access to `DEVICE` which should point to either `cpu` or `cuda:0`